core/src/com/unciv/logic/map/mapgenerator/MapRegions.kt

```diff
@@ -1389,8 +1389,8 @@ class MapRegions (val ruleset: Ruleset){
 
         // Third add some minor deposits to land tiles
         // Note: In G&K there is a bug where minor deposits are never placed on hills. We're not replicating that.
-        val frequency = (baseMinorDepositFrequency * bonusMultiplier).toInt()
-        val minorDepositsToAdd = (landList.size / frequency) + 1 // I sometimes have division by zero errors on this line
+        val frequency = maxOf((baseMinorDepositFrequency * bonusMultiplier).toInt(),1) 
+        val minorDepositsToAdd = (landList.size / frequency) + 1 
```

Comment:
`baseMinorDepositFrequency` is the constant value 30
`bonusMultiplier` can be 1.5, 0.66 or 1
I really can't tell why it would give a divide by zero. Do you have a save-file so I can test?